Praca_domowa_4/Zad_4_8.py

```diff
@@ -0,0 +1,26 @@
+"""Zaimplementuj metodę statyczną tworzącą koszyk na podstawie listy podanych produktów.
+Każdy z nich powinien zostać dodany do koszyka tylko raz.
+
+Przykład użycia:
+backset = Basket.with_products([prod_1, prod_2])"""
+
+
+class Basket:
```

Comment:
Tutaj zadanie może nie jest precyzyjne, ale chodzi o dodanie do klasy `Basket`, którą robiliśmy na zajęciach właśnie tę metodę. No i chodzi o to, żeby ta metoda zwróciła nowy obiekt klasy `Basket`, ale z już dodanymi produktami z listy, którą przekazujesz, tak jakbyś dla każdego produktu zrobiła `koszyk.add_product()`.